src/HeaderRow.tsx

```diff
@@ -25,21 +26,22 @@ export interface HeaderRowProps<R, SR, K extends React.Key> extends SharedDataGr
 }
 
 const headerRow = css`
-  contain: strict;
-  contain: size layout style paint;
-  display: grid;
-  grid-template-columns: var(--rdg-template-columns);
-  grid-template-rows: var(--rdg-header-row-height);
-  height: var(--rdg-header-row-height); /* needed on Firefox */
+  display: contents;
   line-height: var(--rdg-header-row-height);
-  width: var(--rdg-row-width);
-  position: sticky;
-  top: 0;
   background-color: var(--rdg-header-background-color);
   font-weight: bold;
-  z-index: 3;
   outline: none;
```

Comment:
This one as well, don't think we can set the focus on a `display: contents` element
```suggestion
```